lib/src/vo/request/sticker_request.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'sticker_request.g.dart';
+
+@JsonSerializable()
+class StickerRequest with EquatableMixin {
```

Comment:
does this class exported in `lib/mixin_bot_sdk_dart.dart` ?

Reply:
done